src/core.js

```diff
@@ -44,8 +44,10 @@ function createModule( name, testEnvironment ) {
 		name: moduleName,
 		parentModule: parentModule,
 		tests: [],
+		unskippedTests: [],
 		moduleId: generateHash( moduleName ),
 		testsRun: 0,
+		unskippedTestsRun: 0,
```

Comment:
I created a new property because `testsRun` is also referenced [here](https://github.com/qunitjs/qunit/blob/master/src/test.js#L77) and [here](https://github.com/qunitjs/qunit/blob/master/src/test.js#L282). I'm not sure what the first one is doing, but if we change the meaning of `testsRun` then per the second one, if a module ends with skipped tests, wouldn't we log the end of the suite before logging those skipped tests?